src/Symfony/Component/VarDumper/Cloner/Data.php

```diff
@@ -279,57 +279,6 @@ public function dump(DumperInterface $dumper)
     }
 
     /**
-     * @internal
-     */
-    public function serialize()
```

Comment:
and we neither need implementing Serializable anymore: that adds to consumed CPU cycles, and the real issues (GC & size of serialized string) comes from arrays really.